usecases/objects/head.go

```diff
@@ -29,7 +29,7 @@ func (m *Manager) HeadObject(ctx context.Context, principal *models.Principal, c
 	if err := m.authorizer.Authorize(principal, authorization.READ, authorization.Objects(class, tenant, id)); err != nil {
 		return false, &Error{err.Error(), StatusForbidden, err}
 	}
-	if err := m.authorizer.Authorize(principal, authorization.READ, authorization.CollectionsMetadata(class)...); err != nil {
+	if err := m.authorizer.Authorize(principal, authorization.READ, authorization.CollectionsData(class)...); err != nil {
```

Comment:
I think the one above is already checking for Data access, so you can remove it!